content/installation/python/PythonConfig.md

```diff
@@ -11,40 +11,136 @@ The Python agent and Contrast Service use a YAML file to alter the agent behavio
 
 1. The current working directory (e.g., *./contrast_security.yaml*)
-2. A subdirectory called *config*, (e.g., *./config/contrast_security.yaml*)
+2. Within the server's *etc/contrast* directory (e.g., */etc/contrast/contrast_security.yaml*)
 3. Within the server's *etc* directory (e.g., */etc/contrast_security.yaml*)
 
-## General Configuration Options
-
 The configuration YAML consists of four sections. The agent and service may share a common configuration file, but only some options and sections are applicable to each process.
 
-* `contrast`: Options for locating and communicating with the Dashboard of the Contrast interface
-  * `url`: URL to connect to the Contrast application
-  * `api_key`: Organization's API key
-  * `service_key`: Service Key of Organization
-  * `user_name`: Username of user in TeamServer
-* `agent`: Options for communicating between the agent and the service
-  * `logger`:
-    * `path`: Filename of the Contrast Security log file for the agent (*contrast_agent.log*)
-    * `level`: Level of logging details (DEBUG, INFO, WARN, ERROR)
+## Contrast UI Properties
+
+Use the properties in this section to connect the agent to the Contrast UI.
+
+* **contrast**: 
+  * **enable**: Only set this property if you want to turn off Contrast. Set to `true` to turn the agent on; set to `false` to turn the agent off.
+  * **url**: Set the URL for the Contrast UI. **Required.** <br> Example: *https://app.contrastsecurity.com/Contrast*
+  * **api_key**: Set the API key needed to communicate with the Contrast UI. **Required.**
+  * **service_key**: Set the service key needed to communicate with the Contrast UI. It is used to calculate the Authorization header. **Required.**
+  * **user_name**: Set the user name used to communicate with the Contrast UI. It is used to calculate the Authorization header. **Required.**
+  * **last_config_path**: Set the path to which the agent should store the currently used configuration from the Contrast UI. <br> Example: *./tmp/config*
+
+## Contrast Agent Properties
+
+### Logger
+
+Define the following properties to set logging values. If these properties aren't defined, the agent uses the logging values from the Contrast UI.
+
+* **agent**: Options for communicating between the agent and the service
+  * **logger**:
+    * **path**: Enable diagnostic logging by setting a path to a log file. While diagnostic logging hurts performance, it generates useful information for debugging Contrast. The value set here is the location to which the agent saves log output. If no log file exists at this location, the agent creates a file. <br> Example - */opt/Contrast/contrast.log* creates a log in the */opt/Contrast* directory, and rotates it automatically as needed.
+    * **level**: Set the the log output level. Value options are `OFF`, `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`, `TRACE`, and `ALL`.
     * `progname`: Name used to identify the process within the log file (Contrast Agent)
```

Comment:
@palmergs I don't see this property listed in the YAML template for Python. Has this been added very recently?

Reply:
`progname` is used in the python agent. Are you saying it was missed in this update?